docs/api/java/getting-started.md

```diff
@@ -422,402 +441,542 @@ public class Main {
+import io.zksync.protocol.ZkSync;
+import io.zksync.protocol.provider.EthereumProvider;
+import org.web3j.crypto.Credentials;
+import org.web3j.protocol.Web3j;
 import org.web3j.protocol.core.methods.response.TransactionReceipt;
+import org.web3j.tx.RawTransactionManager;
+import org.web3j.tx.TransactionManager;
+import org.web3j.tx.gas.ContractGasProvider;
+import org.web3j.tx.gas.StaticGasProvider;
 
 import java.math.BigInteger;
 
 public class Main {
-    public static void main(String... args) {
-        ZkSyncWallet wallet; // Initialize wallet
-
-        BigInteger amount = Token.ETH.toBigInteger(0.5);
-
-        // ETH By default
-        TransactionReceipt receipt = wallet.withdraw("0x<receiver_address>", amount).send();
-
-        // Also we can withdraw ERC20 token
-        Token token;
-
-        TransactionReceipt receipt = wallet.withdraw("0x<receiver_address>", amount, token).send();
-    }
+  public static void main(String... args) {
+    Web3j web3j; //Initialize web3j
+    ZkSync zksync; //Initialize zksync
+    Credentials credentials; //Initialize credentials
+    BigInteger chainId; //Initialize chainId
+
+    TransactionManager manager = new RawTransactionManager(web3j, credentials, chainId.longValue());
+    BigInteger gasPrice = web3j.ethGasPrice().send().getGasPrice();
+    ContractGasProvider gasProvider = new StaticGasProvider(gasPrice, BigInteger.valueOf(300_000L));
+    TransactionReceipt receipt = EthereumProvider
+      .load(zksync, web3j, manager, gasProvider).join()
+      .finalizeWithdraw("TRANSACTION_HASH", 0);
+  }
 }
-
 ```
 
-## Wallet
+### Calculate transaction fees
 
-Get the price of the execution of the transaction
+#### Get price of the transaction execution (currently not wokring properly)
 
-```java
+::: warning
+Feature currently unsupported. Under development
+:::
 
-import io.zksync.crypto.signer.EthSigner;
+```java
 import io.zksync.methods.request.Transaction;
-import io.zksync.protocol.zksync;
+import io.zksync.protocol.ZkSync;
 import io.zksync.transaction.fee.Fee;
 
+import java.io.IOException;
 import java.math.BigInteger;
 
 public class Main {
-    public static void main(String... args) {
-        ZkSync zksync; // Initialize client
+  public static void main(String... args) throws IOException {
+    ZkSync zksync; // Initialize client
 
-        Transaction forEstimate; // Create transaction as described above
+    Transaction forEstimate; // Create transaction as described above
 
-        Fee fee = zksync.zksEstimateFee(forEstimate).send().getResult();
+    Fee fee = zksync.zksEstimateFee(forEstimate).send().getResult();
 
-        // Also possible to use eth_estimateGas
-        BigInteger gasUsed = zksync.ethEstimateGas(forEstimate).send().getAmountUsed();
-    }
+    // Also possible to use eth_estimateGas
+    BigInteger gasUsed = zksync.ethEstimateGas(forEstimate).send().getAmountUsed();
+  }
 }
-
 ```
 
 ### Get fee via TransactionFeeProvider
```

Comment:
if this method of estimating a fee is working properly, let's move it before the other one that currently does not work.